packages/graphics/src/GraphicsGeometry.ts

```diff
@@ -51,184 +51,108 @@ export class GraphicsGeometry extends BatchGeometry
+     * */
+    points: Array<number> = [];
 
-        /**
-         * An array of points to draw, 2 numbers per point
-         *
-         * @member {number[]}
-         * @protected
-         */
-        this.points = [];
-
-        /**
-         * The collection of colors
-         *
-         * @member {number[]}
-         * @protected
-         */
-        this.colors = [];
-
-        /**
-         * The UVs collection
-         *
-         * @member {number[]}
-         * @protected
-         */
-        this.uvs = [];
-
-        /**
-         * The indices of the vertices
-         *
-         * @member {number[]}
-         * @protected
-         */
-        this.indices = [];
-
-        /**
-         * Reference to the texture IDs.
-         *
-         * @member {number[]}
-         * @protected
-         */
-        this.textureIds = [];
-
-        /**
-         * The collection of drawn shapes.
-         *
-         * @member {PIXI.GraphicsData[]}
-         * @protected
-         */
-        this.graphicsData = [];
-
-        /**
-         * Used to detect if the graphics object has changed.
-         *
-         * @member {number}
-         * @protected
-         */
-        this.dirty = 0;
-
-        /**
-         * Batches need to regenerated if the geometry is updated.
-         *
-         * @member {number}
-         * @protected
-         */
-        this.batchDirty = -1;
-
-        /**
-         * Used to check if the cache is dirty.
-         *
-         * @member {number}
-         * @protected
-         */
-        this.cacheDirty = -1;
-
-        /**
-         * Used to detect if we cleared the graphicsData.
-         *
-         * @member {number}
-         * @default 0
-         * @protected
-         */
-        this.clearDirty = 0;
-
-        /**
-         * List of current draw calls drived from the batches.
-         *
-         * @member {object[]}
-         * @protected
-         */
-        this.drawCalls = [];
-
-        /**
-         * Intermediate abstract format sent to batch system.
-         * Can be converted to drawCalls or to batchable objects.
-         *
-         * @member {PIXI.graphicsUtils.BatchPart[]}
-         * @protected
-         */
-        this.batches = [];
-
-        /**
-         * Index of the last batched shape in the stack of calls.
-         *
-         * @member {number}
-         * @protected
-         */
-        this.shapeIndex = 0;
+    /**
+     * The collection of colors
+     *
+     * @member {number[]}
+     * */
+    colors: Array<number> = [];
 
-        /**
-         * Cached bounds.
-         *
-         * @member {PIXI.Bounds}
-         * @protected
-         */
-        this._bounds = new Bounds();
-
-        /**
-         * The bounds dirty flag.
-         *
-         * @member {number}
-         * @protected
-         */
-        this.boundsDirty = -1;
+    /**
+     * The UVs collection
+     *
+     * @member {number[]}
+     * */
+    uvs: Array<number> = [];
```

Comment:
Why not using `number[]` ? Same in multiple places here.

Reply:
I remember @ivanpopelyshev wanted to create some kinda convention with `Array<T>` vs `T[]`, but I don't think we've been following it or have settled on a convention.

Overall, if we can use the same type to remove more documentation, that would be helpful, otherwise I don't think it's that critical.

Reply:
I think the idea was that arrays with dynamic length should be `Array<T>`, while read-only or fixed-length ones should use `T[]`.

Reply:
I'll explore these options and resulting documentation html and post the result.